webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiConversationsSettings.js

```diff
@@ -82,6 +81,14 @@ export class SakaiConversationsSettings extends SakaiElement {
     return this._i18n && this.settings;
   }
 
+  updated(changedProperties) {
+    if (changedProperties.has("settings") && this.settings) {
+      this._guidelines = (this.settings.guidelines && this.settings.guidelines.trim())
+        ? this.settings.guidelines
+        : (this._i18n && this._i18n.community_guidelines_sample);
```

Comment:
shouldUpdate should ensure this is set. You should not need this guard.

Reply:
done